terraform/workspace-variables/app_config.yml

```diff
@@ -36,6 +36,7 @@ development:
 # ey-recovery-content
 content:
   <<: *defaults
+  GCS_BUCKET_NAME: 'pr-review'
```

Comment:
where are the other bucket names being set? they don't need to be encrypted in the credentials though

Reply:
@mo-zag you've set the bucket name in the development creds to be `pr-review` too. You'll need to document the buckets because I presume this was just a temporary choice to expedite things.